spec/haskell/src/SEL4/Object/Reply.lhs

```diff
@@ -28,6 +28,11 @@ This module specifies the behavior of reply objects.
 
 \end{impdetails}
 
+> getReply :: PPtr Reply -> Kernel Reply
+> getReply rptr = getObject rptr
+
+> setReply :: PPtr Reply -> Reply -> Kernel ()
+> setReply rptr r = setObject rptr r
```

Comment:
Do we need to move this?

Reply:
Yeah, it's not needed but it made sense to me to have them next to `updateReply`, since that's now the main place that they're used. I'm happy to undo it if you'd prefer not.

Reply:
Ah, I see. That makes sense! No need to move, happy for them to stay here.